me.go

```diff
@@ -1,27 +1,40 @@
 package control
 
+// The Me struct represents data accociated with your Ably account.
```

Comment:
It is actually "information about the [token](https://ably.com/documentation/control-api#authentication) the current user authenticates with.", not "data associated with account"

Reply:
https://ably.com/docs/api/control-api#tag/tokens/paths/~1me/get

Reply:
Yeah thanks. Was a but difficult to figure out how to label the different me components.

Reply:
Should we update the comment?